### PR TITLE
Implement queries and update animals

### PR DESCRIPTION
**Pull request by benshidanny11**
## In this milestone I:
- :heavy_check_mark: Inserted extra data
- :heavy_check_mark: Updated animal table using transaction commands
- :heavy_check_mark: Added column species to table animals
- :heavy_check_mark: Fetched data conditionally

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -13,4 +13,45 @@ SELECT date_of_birth FROM animals WHERE name='Agumon' OR name='Pikachu';
...
+
+ UPDATE animals SET weight_kg=weight_kg*-1 where weight_kg<0;
+
+ SELECT count(*) FROM animals;
+
+SELECT count(*) FROM animals WHERE escape_attempts=0;
+
+SELECT AVG(weight_kg) FROM animals;
+SELECT MAX(escape_attempts) FROM animals WHERE neutered=true OR neutered=false;
+SELECT MAX(weight_kg),MIN(weight_kg) FROM animals WHERE species='digimon';
```
Comment (BenMukebo) [inline queries.sql]: - [ ] Great job so far however you did not write queries to answer the following questions:

- What is the minimum and maximum weight of each type of animal?
eg 
```sql
  SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals
   GROUP BY species;
```
- What is the average number of escape attempts per animal type of those born between 1990 and 2000?
 eg :
```sql
   SELECT species, AVG(escape_attempts) FROM animals
   GROUP BY species WHERE  date_of_birth >= '1990/01/01' AND ...';
```

- Kindly make this and add a screenshots of their queries results to the pull request description 📸

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @benshidanny11,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (benshidanny11) [reply queries.sql]: @BenMukebo, thanks for providing me with feedback. I am going to work on them :ok_hand:

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @benshidanny11,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**